pykern/quest.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""API wrapper
+
+:copyright: Copyright (c) 2022 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkcollections import PKDict
+from pykern.pkdebug import pkdc, pkdlog, pkdp
+
+
+class Args:
+    # qargs
+    pass
+
+
+class API(PKDict):
+    # qcall
```

Comment:
This is the only one not like the others. Name the class Call?

Reply:
The instance is a call. The class is an API. I know if it is "confusing", but it's something we can get used to. The semantics are important. You cannot say API().api_beginRequest (as is done on master), because you have to qcall.call_api("beginRequest") to call that method. However, all we have is the concept of "self", which is the qcall, which allows the API during a  call to call useful methods, e.g. self.reply_ok. These utilities are not APIs. I thought it was prudent to not have to say self.qcall.reply_ok. Perhaps that is a mistake.